ui/editor/src/ctrl.js

```diff
@@ -12,6 +12,7 @@ module.exports = function(cfg) {
   this.trans = lichess.partial(editor.trans, this.data.i18n);
 
   this.vm = {
+    draggingSpare: m.prop(false),
```

Comment:
do we really need the extra state? Can't we use `chessground.state.draggable.current.newPiece` instead?